.circleci/early_exit.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+commitmessage=$(git log --pretty=%B -n 1)
+if [[ $commitmessage = *"[ci skip]"* ]]; then
```

Comment:
Travis supports both `[skip ci]` and `[ci skip]`. Personally I prefer, `[skip ci]`. Shouldn't we support both?